flake.nix

```diff
@@ -165,5 +165,16 @@
           ];
       });
     };
+
+    hydraJobs = with nixpkgs.lib;
+      let
+        hydraJobs = mapAttrs (_: hydraJob);
+      in {
+        checks = hydraJobs self.checks.x86_64-linux;
+        packages = hydraJobs self.packages.x86_64-linux;
```

Comment:
This breaks `nix flake check`: https://github.com/internet4refugees/beherbergung/pull/47/commits/03168970613f9375a9338a26396fdd55dd69ded9